MediaBrowser.Model/Dlna/StreamBuilder.cs

```diff
@@ -151,25 +142,21 @@ private StreamInfo[] SortMediaSources(List<StreamInfo> streams, long maxBitrate)
 
                 return 0;
 
-            }).ThenBy(streams.IndexOf).ToArray();
+            }).ThenBy(streams.IndexOf);
         }
 
-        private TranscodeReason? GetTranscodeReasonForFailedCondition(ProfileCondition condition)
+        private static TranscodeReason? GetTranscodeReasonForFailedCondition(ProfileCondition condition)
         {
             switch (condition.Property)
             {
-                case ProfileConditionValue.AudioBitrate:
-                    if (condition.Condition == ProfileConditionType.LessThanEqual)
-                    {
+                case ProfileConditionValue.AudioBitrate when condition.Condition == ProfileConditionType.LessThanEqual:
                         return TranscodeReason.AudioBitrateNotSupported;
```

Comment:
Indentation is now off, plus what's the reason to check for condition if both `case` statements return same reason?